src/schema/submission.yaml

```diff
@@ -379,6 +378,9 @@ classes:
       - individual
     slot_usage:
       name:
+        description: >-
+          A descriptive name of the Biospecimen (e.g. GHGAB_caudate_nucleus_biospecimen).
+          Please do not enter any personally identifiable data!
```

Comment:
Same suggestion as the previous similar case (directly addressing the user at the documentation page)